src/GetterSetter.php

```diff
@@ -10,63 +10,74 @@
 
 use Revinate\GetterSetter as gs;
 
-class GetterSetter implements GetterSetterInterface {
+class GetterSetter implements GetterSetterInterface
+{
 
     protected $pathSeparator = '.';
 
     /**
      * @description Gets the value for the field $fieldName.
      * @param array|object $data
-     * @param string       $fieldName
-     * @param null         $default
+     * @param string $fieldName
+     * @param null $default
      * @return mixed
+     * @throws UnableToGetFieldException
      */
-    public function getValue($data, $fieldName, $default = null) {
+    public function getValue($data, $fieldName, $default = null)
+    {
         return gs\getValue($data, $fieldName, $default);
     }
 
     /**
      * @description Sets the value for the field $fieldName
      * @param array|object $data
-     * @param string       $fieldName
-     * @param mixed        $value
+     * @param string $fieldName
+     * @param mixed $value
      * @return $this
+     * @throws UnableToSetFieldException
      */
-    public function setValue($data, $fieldName, $value) {
+    public function setValue($data, $fieldName, $value)
+    {
```

Comment:
This is not our style of brackets, it should be on the same line as the function.